docs/index_whyte.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: page
+---
+# Build a User Task Management System with *To Do*
+
+Connect our database to the apps your customers use every day
+
+## The basics
+
+To Do is a cloud service that helps your customers stay on track by generating task reminders. The lightweight API provides access to To Do data and functionality, allowing developers to build custom features and branded interfaces. The To Do API uses REST and returns HTTP response codes and responses encoded in JSON.
+
+There are two resources.
+
+* **Users** - Individuals who sign up for your To Do service. Each user can have multiple tasks. The Users resource includes endpoints to create, retrieve, update, and delete user information.
+
+* **Tasks** - Save To Do tasks here. Each task has details such as title, description, and due date. The Tasks resource includes endpoints to create, retrieve, update, and delete tasks.
+
+## Benefits
+
+* **Brandable** - Create a bespoke front end that syncs with your app and corporate branding.
+
+* **Lightweight** - Registered users drive data input with easy-to-read forms, so expect a minimal impact on your customer support team.
+
+* **Customer-focussed** - Help your customers succeed and give them another reason to come back to your app and site.
+
+## Getting help
+
+Contact the [tech support](mailto:support@yopmail.com) or visit the [development portal](https://example.com/) to learn more.
+
+## Support documentation
+
+Sure, we have To Do docs.
+
+* **Tutorials** - Guided instructions that show you how to use the service. See [below](#tutorials).
```

Comment:
```suggestion
* **Tutorials** - Guided instructions that show you how to use the service. Check out [the complete list of tutorials](#tutorials).
```
Link text should describe the destination to inform the reader what they'll find when they get there.
"See" implies having vision. Visit, check out, open, etc. don't make that assumption.